src/stores/login/index.ts

```diff
@@ -0,0 +1,55 @@
+import {makeAutoObservable} from 'mobx';
+import {propertyStore} from "../index";
+import {hash} from "@0auth/message";
+
+type LoginType = {
+  isLogin: boolean;
+  password: string;
+}
+
+class Login {
+  isLogin: boolean = false;
+  password: string = '';
+  hash: string = '';
+
```

Comment:
`isLogin`: variable to check if you are currently logged in.
`password`: user's password input
`hash`: hash of password